src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala

```diff
@@ -107,6 +107,18 @@ trait UserApiService
       }
     } ~
     pathPrefix("api") {
+      pathPrefix("profile" / "billing") {
```

Comment:
Shouldn't you remove the incorrect block from the last PR up at line 97?

Reply:
Yes, I should.